Market-API.txt

```diff
@@ -9,7 +9,13 @@ The current on that is:
 
 from tribler/Tribler/community/market/community.py
 
-Division of responsibility: the market is fully responsible for the exchange of funds and management of the wallets of both Multi-chain, Bitcoin.
+Division of responsibility: 
+The Multi-chain database, wallet and money transfer mechanism is fully under control of The Market team.
+A get_balance() function for multi-chain needs to be created by either: one of the BEP teams of dev team of multi-chain.
```

Comment:
Since the Multi-Chain database, wallet and money are already in the domain of the Market team, it makes no sense if the get_balance() (which already needs to be implemented for a working wallet) is the domain of the autonomous self-replicating code